nova_vm/src/engine/bytecode/executable.rs

```diff
@@ -69,16 +69,43 @@ impl CompileContext<'_> {
     }
 }
 
+#[derive(Debug)]
+/// A `Send` and `Sync` wrapper over a `&'static T` where `T` might not itself
+/// be `Sync`. This is safe because the reference can only be obtained from the
+/// same thread in which the `SendableRef` was created.
+pub(crate) struct SendableRef<T: 'static> {
```

Comment:
**thought:** I definitely would've just gone with "sure sure we don't deref this outside the main thread so it's all safe" :D but this is much better